modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -109,9 +109,16 @@ listen my-cluster-api-6443
   http-check connect
   http-check send meth GET uri /readyz
   http-check expect status 200
+ifndef::bare-metal[]
     server my-cluster-master-2 192.168.1.101:6443 check inter 10s rise 2 fall 2
     server my-cluster-master-0 192.168.1.102:6443 check inter 10s rise 2 fall 2
     server my-cluster-master-1 192.168.1.103:6443 check inter 10s rise 2 fall 2
+endif::bare-metal[]
+ifdef::bare-metal[]
```

Comment:
Note that this isn't specific to baremetal. Cross-subnet loadbalancers can be used with any on-prem platform, and baremetal users may choose not to use a cross-subnet control plane but still run their own LB.

I'm not entirely sure how to handle that though. Maybe we need to factor this config out and include the appropriate one for each scenario?